mmv1/templates/terraform/update_encoder/region_ssl_policy.erb

```diff
@@ -0,0 +1,25 @@
+<%# The license inside this block applies to this file.
+	# Copyright 2017 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+// copy from ssl_policy.erb to handle fingerprint during updates
+// TODO(https://github.com/GoogleCloudPlatform/magic-modules/issues/184): Handle fingerprint consistently
+obj["fingerprint"] = d.Get("fingerprint")
```

Comment:
i'm not super familiar with `fingerprint` but i also saw this available that you can maybe add to `terraform.yaml`
`fingerprint_name`

Reply:
Looks like `fingerprint_name` can be set in `api.yaml`, there is also a dedicated type to try out `- !ruby/object:Api::Type::Fingerprint`